### PR TITLE
Tg2 56 board page get data from redux

### PR DESCRIPTION
**Pull request by salitordjman**
[TG2-56](https://appleseedsstudents.atlassian.net/browse/TG2-56)

### CODE REVIEW

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
+import { useEffect } from "react";
+
+import myApi from "../../../../../Url/Url";
 
 function PageBoard() {
+    useEffect(() => {
+        getTasksByIdProject("blabla")
+    }, []); 
+
+    const getTasksByIdProject =async (idProject:string) => {
```
Comment (Toam10) [inline frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx:11]: If you use it only in the useEffect please writ it inside the useEffect we dont want unpredict use cases

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
+        try {
+            const result = await Promise.all([
```
Comment (Toam10) [inline frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx:13]: please use an [distracturing] fot the results

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx [hunk 3]
```diff
@@ -1,11 +1,34 @@
...
+                myApi.get(`/project/fetchProjectDetails/6220a25b4051ee90bc98bad6`), 
+                myApi.get(`task/fetchTaskDetailsByIdProject/6220a25b4051ee90bc98bad6`)
+            ]);
+
+            //   const resultProject = await myApi.get(`/project/fetchProjectDetails/${idProject}`);
+            //   const result = await myApi.get(`/profile/user/${idProject}`);
+            const resultProject=result[0].data.labels;
+            const resultTasks=result[1].data;
+            console.log(resultProject);
```
Comment (Toam10) [inline frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx:22]: please remove console.log()

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx [hunk 4]
```diff
@@ -1,11 +1,34 @@
...
+    const getTasksByIdProject =async (idProject:string) => {
+        try {
+            const result = await Promise.all([
+                myApi.get(`/project/fetchProjectDetails/6220a25b4051ee90bc98bad6`), 
+                myApi.get(`task/fetchTaskDetailsByIdProject/6220a25b4051ee90bc98bad6`)
+            ]);
+
+            //   const resultProject = await myApi.get(`/project/fetchProjectDetails/${idProject}`);
+            //   const result = await myApi.get(`/profile/user/${idProject}`);
+            const resultProject=result[0].data.labels;
```
Comment (Toam10) [inline frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx:20]: remove it to the distracture data

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx [hunk 5]
```diff
@@ -1,11 +1,34 @@
...
+            const resultTasks=result[1].data;
```
Comment (Toam10) [inline frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx:21]: remove it to the distracture data

File: frontend/src/App.tsx [hunk 6]
```diff
@@ -1,14 +1,26 @@
...
+      <Router>
       <Navbar />
-      <PageDesign />
+      
+      <Routes>
+
+      <Route path="/" element={<PageDesign />} />
+      <Route path="board/:idProject" element={<PageBoard />} />
+
+
```
Comment (Toam10) [inline frontend/src/App.tsx:21]: remove empty line spaces

File: frontend/src/App.tsx [hunk 7]
```diff
@@ -1,14 +1,26 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+
 import './App.css';
 import PageDesign from './Screens/GlobalComponents/Navbar/Components/PageDesign/PageDesign';
 import './Screens/GlobalComponents/Navbar/Navbar';
```
Comment (Toam10) [inline frontend/src/App.tsx:6]: remove it if you dont use it

File: frontend/src/App.tsx [hunk 8]
```diff
@@ -1,14 +1,26 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+
 import './App.css';
```
Comment (Toam10) [inline frontend/src/App.tsx:4]: take it to the bottom of the imports this is a .css file

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx [hunk 9]
```diff
@@ -0,0 +1,13 @@
+import './PageBoard.css';
+
+function PageBoard() {
+  return (
+      <div>
```
Comment (Toam10) [inline frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx:30]: remove unused div

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+          <div>
```
Comment (Toam10) [inline frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx]: remove unused div

**Review by Toam10 — APPROVED** [review]

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx (hunk 1 above)
Comment (salitordjman) [reply frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx:11]: i was change it in the future branch

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx (hunk 2 above)
Comment (salitordjman) [reply frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx:13]: I was change it in the future branch

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx (hunk 4 above)
Comment (salitordjman) [reply frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx:20]: I was change it in the future branch

File: frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx (hunk 5 above)
Comment (salitordjman) [reply frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx:21]: I was change it in the future branch

**Comment by salitordjman** [comment]
[TG2-56](https://appleseedsstudents.atlassian.net/browse/TG2-56)

**Pull request merged**